application/modules/admin/views/scripts/response/edit-tb-response.phtml

```diff
@@ -44,11 +43,11 @@ if (isset($this->responseData['shipment']["shipment_test_report_date"]) &&
         </td>
         <td>
             <select id="notTestedReason" name="notTestedReason" class="form-control" title="Please select reason" style="width:60%;">
-                <option value="">--Select--</option>
+                <option selected>--Select--</option>

```

Comment:
Please set a value attribute on these "--Select--" options: `value=""`. When testing this I'm observing that the assay selected is being saved as "--Select--" in shipment_participant_map.attributes like this:
```
{
	"cartridge_lot_no": "54915",
	"expiry_date": "2020-03-15",
	"assay": "--Select--",
	"count_tests_conducted_over_month": "251",
	"count_errors_encountered_over_month": "2",
	"error_codes_encountered_over_month": "5007"
}
```